foxglove/cmd/pending_imports.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/foxglove/foxglove-cli/foxglove/console"
+	"github.com/spf13/cobra"
+)
+
+func newPendingImportsCommand(params *baseParams) *cobra.Command {
+	var format string
+	var requestId string
+	var deviceId string
+	var deviceName string
+	var filename string
+	var error string
+	var showCompleted bool
+	var showQuarantined bool
+	var siteId string
+	pendingImportsCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List the pending imports. These are in-progess import jobs for newly uploaded recordings.",
```

Comment:
These may also include errors, not just in-progress. But we also want to keep it short; here's one suggestion.

```suggestion
		Short: "List pending and errored import jobs for uploaded recordings.",
```